Assets/MRTK/Core/Inspectors/Profiles/MixedRealityControllerVisualizationProfileInspector.cs

```diff
@@ -192,7 +195,7 @@ private void RenderControllerList(SerializedProperty controllerList)
                 {
                     controllerList.DeleteArrayElementAtIndex(i);
                     EditorGUILayout.EndHorizontal();
-                    GUILayout.EndVertical();
+                    EditorGUILayout.EndVertical();
```

Comment:
Am I missing something, or is there no corresponding `EditorGUILayout.BeginVertical();` call?

Reply:
Yeah this was vestigial code that I tried to leave in unchanged but it looks like I inadvertently made an edit to it anyways. I'll delete both since they don't have a corresponding begin vertical call